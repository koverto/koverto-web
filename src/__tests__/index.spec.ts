import { act } from "@testing-library/react"

describe("entrypoint", () => {
  it("renders the app container", () => {
    act(() => {
      require("../index")
    })
    const app = document.getElementById("app")

    expect(app).not.toBeNull()
    expect(app).toMatchSnapshot()
  })
})
