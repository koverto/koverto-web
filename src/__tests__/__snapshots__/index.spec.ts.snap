// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`entrypoint renders the app container 1`] = `
<div
  id="app"
>
  <div>
    <h1>
      Login
    </h1>
    <form>
      <div>
        <input
          data-testid="email"
          name="email"
          type="email"
        />
      </div>
      <div>
        <input
          data-testid="password"
          name="password"
          type="password"
        />
      </div>
      <button
        data-testid="submit"
        type="submit"
      >
        Login
      </button>
    </form>
  </div>
</div>
`;
